src/components/NotFoundPage/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Helmet from 'react-helmet';
```

Comment:
nit: The docs show `Helmet` as being a named export that should be imported like `import { Helmet }`